### ISSUE TITLE
Login failed

### ISSUE THREAD

**Issue by sp3rial** [fineemb/Lynk-Co#4]
Can't wait to try this integration, but the login is failing. Anything to deal with the code request at login maybe?

**Comment by robbinonline** [comment]
@fineemb Hi any changes you can take a look at the login problem?

**Comment by albamatti** [comment]
@fineemb Hi something wrong with login. Can we help?

**Comment by robbinonline** [comment]
I found out this repo is not gonna work in Europe. This is based on Linux OS in europa it's a closed android OS.

**Comment by sp3rial** [comment]
> I found out this repo is not gonna work in Europe. This is based on Linux OS in europa it's a closed android OS.

Yes this is true. Unfortunately at the moment it's not possible to use this addon to integrate Lynk&CO with homeassistant. There's also a thread on the Lynk developer page, but I haven't seen any interest from the Lynk team.

**Comment by timdows** [comment]
Did somebody find a way to (in Europe) connect to the API endpoint of Lynk&Co?

I'm diving into the options to locate my car every hour (or random interval) and get this information via the same way the app is doing
I will look into the app how it is connecting to the back-end systems, but havent find a solution yet

**Comment by fineemb** [comment]
This is an old API. Officials may abandon it at any time. Not suitable for long-term use. In addition, querying too frequently will deteriorate the vehicle battery

**Comment by timdows** [comment]
@fineemb Thanks for the reply!

I would have guessed that the API we could query (the one the android/ios app is using) will be on the cloud somewhere
I have been looking for the API endpoint and how the android app connects to it

**Comment by TobiasLaross** [comment]
European version: https://github.com/TobiasLaross/Hass-Lynk-Co

**Comment by timdows** [comment]
> European version: https://github.com/TobiasLaross/Hass-Lynk-Co

Just quickly tested it, and it connects and gets data
![image](https://github.com/fineemb/Lynk-Co/assets/9577532/53f45b38-17df-49ad-b45c-d0ef2aa455fe)

Really nice at first glance!
